### PR TITLE
Litani port

### PR DESCRIPTION
**Pull request by markrtuttle**
This pull request ports the CBMC HTTP proofs to the litani build system so we can use the wellpsring implementation of continuous integration with HTTP.

### CODE REVIEW

File: test/cbmc/.gitignore [hunk 1]
```diff
@@ -11,3 +11,9 @@ TAGS-*
 arpa_cmake/
 arpa-validation-logs/
 Makefile.arpa
+
+# Emitted by litani
+.litani_cache_dir
+.ninja_deps
+.ninja_log
+*~
```
Comment (karkhaz) [inline test/cbmc/.gitignore:19]: newline

File: .gitmodules [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
 [submodule "test/unit-test/CMock"]
-	path = test/unit-test/CMock
-	url = https://github.com/ThrowTheSwitch/CMock.git
+        path = test/unit-test/CMock
+        url = https://github.com/ThrowTheSwitch/CMock.git
 [submodule "source/third-party/http_parser"]
-	path = source/third-party/http_parser
-	url = https://github.com/nodejs/http-parser.git
+        path = source/third-party/http_parser
+        url = https://github.com/nodejs/http-parser.git
```
Comment (karkhaz) [inline .gitmodules:9]: gratuitous whitespace changes

**Review by karkhaz — APPROVED** [review]

File: test/cbmc/.gitignore (hunk 1 above)
Comment (markrtuttle) [reply test/cbmc/.gitignore]: Fixed.

File: .gitmodules (hunk 2 above)
Comment (markrtuttle) [reply .gitmodules]: Fixed.

**Review by karkhaz — APPROVED** [review]

**Pull request closed without merging**